picard/config.py

```diff
@@ -249,21 +249,26 @@ def get(cls, section, name):
         return cls.registry.get((section, name))
 
 
+def _convert_to_bool(value):
+    # The QSettings IniFormat saves boolean values as the strings "true"
+    # and "false". Thus, explicit boolean and string comparisons are used
+    # to determine the value. NOTE: In PyQt >= 4.8.3, QSettings.value has
+    # an optional "type" parameter that avoids this. But we still support
+    # PyQt >= 4.5, so that is not used.
```

Comment:
This comment is not valid anymore?

Reply:
I just address the pylint warning here, imho the comment is still valid (we don't use QSettings.value "type"), but we also don't use PyQt4 anymore, so dunno.
Is  there an issue here i didn't see ?

Reply:
I agree, the comment is still valid. Since we are now on newer Qt versions we should look into this and see how we can simplify bool handling here, but that's for a separate PR.